starknet_py/tests/e2e/client/conftest.py

```diff
@@ -207,13 +210,17 @@ def client(request) -> Client:
 
 @pytest_asyncio.fixture(name="prepare_network", scope="module", autouse=True)
 async def fixture_prepare_network(
-    network: str, new_gateway_account_client: AccountClient
+    network: str,
+    new_gateway_account_client: AccountClient,
+    details_of_account_to_be_deployed: AccountToBeDeployedDetails,
```

Comment:
There's a problem with `details_of_account_to_be_deployed` fixture: it has a scope `module` and even though it generates a random salt and key, it's not rerun for every tests - if one attempts to do multiple deploy_account transactions in one test module, all but first will fail. 

This is problematic and will need to be addressed in separate PR.

Reply:
Should be partially resolved now. `details_of_account_to_be_deployed` still isn't re-run, but one can use functions from utils to generate new details.